src/lib.rs

```diff
@@ -543,6 +543,8 @@ impl GameId {
 
 #[cfg(test)]
 mod tests {
+    use std::io::{Read, Write};
```

Comment:
Why this is needed?

Reply:
That was not meant to be included.... That was a part of a test, but I have forgotten to take it out, I will remove it, and resubmit.